site/docs/core/unittest.md

```
@@ -752,7 +773,4 @@ describe('GET /httpclient', () => {
 
 All sample code can be found in [eggjs/exmaples/unittest](https://github.com/eggjs/examples/blob/master/unittest)
 
-[mocha]: https://mochajs.org
-[co-mocha]: https://github.com/blakeembrey/co-mocha
-[nyc]: https://github.com/istanbuljs/nyc
-[power-assert]: https://github.com/power-assert-js/power-assert
+[vitest]: https://vitest.dev
```

markdownlint-cli2 (0.21.0)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "vitest"

(MD053, link-image-reference-definitions)